py/server/deephaven/table.py

```diff
@@ -2049,25 +2050,31 @@ def format_data_bar(self, column: str, value_column: str = None, min: float = No
             if value_column is None:
                 value_column = column
 
+            if isinstance(min, str) and isinstance(max, float):
+                max = value_column
+
+            if isinstance(max, str) and isinstance(min, float):
+                min = value_column
```

Comment:
This was for the scenario that the user just passes in one column for either min or max and leaving the other undefined. The default value is NULL_DOUBLE so I was just over-writing that and assigning the value column so that both min and max will be strings if one is defined as a string.

I'm not sure if this is the desired behaviour though if one of them is defined as a column. If we wanted to keep the other param as a constant I can create 4 overloaded methods for formatDataBar

```
formatDataBar(double min, double max)
formatDataBar(str min, double max)
formatDataBar(double min, str max)
formatDataBar(str min, str max)
```

Reply:
Just spoke with web team and I think this is the intended behaviour so I will add those additional overloaded methods